### ISSUE TITLE
Problem: blockchain regression fails on listed blocks with error "cannot assign requested address"

### ISSUE THREAD

**Issue by sjmackenzie** [ETCDEVTeam/sputnikvm#189]
```
$ RUST_BACKTRACE=1 RUST_LOG=regtests cargo run --bin regtests -- -r http://127.0.0.1:8545 -n 200051..1000000
    Finished dev [unoptimized + debuginfo] target(s) in 0.0 secs
     Running `/home/stewart/dev/fractalide/etcdev/sputnikvm/target/debug/regtests -r 'http://127.0.0.1:8545' -n 200051..1000000`
net version: 1
block 200051, transaction count: 3

working on transaction 0x064b64cff3c6dba8194184cde09981b53bfb005ff5c0fc6cf9475957dd5e2cb1
Feeding VM account code at 0x32be343b94f860124dc4fee278fdcbd38c102d88 ...
VM exited with ExitedOk.

working on transaction 0x66b6c9c9b43781ec99bfd1232280735d35ed95f71247cedfdc21c62350140fda
Feeding VM account code at 0xbbda93a81b882774e1c9b5e3f933fa35dac8bc88 ...
Feeding VM account code at 0xbbda93a81b882774e1c9b5e3f933fa35dac8bc88 ...
Feeding VM account code at 0xbbda93a81b882774e1c9b5e3f933fa35dac8bc88 ...
Feeding VM account code at 0xbbda93a81b882774e1c9b5e3f933fa35dac8bc88 ...
Feeding VM account code at 0xbbda93a81b882774e1c9b5e3f933fa35dac8bc88 ...
Feeding VM account code at 0xbbda93a81b882774e1c9b5e3f933fa35dac8bc88 ...
Feeding VM account code at 0xbbda93a81b882774e1c9b5e3f933fa35dac8bc88 ...
Feeding VM account code at 0xbbda93a81b882774e1c9b5e3f933fa35dac8bc88 ...
Feeding VM account code at 0xbbda93a81b882774e1c9b5e3f933fa35dac8bc88 ...
Feeding VM account code at 0xbbda93a81b882774e1c9b5e3f933fa35dac8bc88 ...
Feeding VM account code at 0xbbda93a81b882774e1c9b5e3f933fa35dac8bc88 ...
Feeding VM account code at 0xbbda93a81b882774e1c9b5e3f933fa35dac8bc88 ...
Feeding VM account code at 0xbbda93a81b882774e1c9b5e3f933fa35dac8bc88 ...
Feeding VM account code at 0xbbda93a81b882774e1c9b5e3f933fa35dac8bc88 ...
Feeding VM account code at 0xbbda93a81b882774e1c9b5e3f933fa35dac8bc88 ...
Feeding VM account code at 0xbbda93a81b882774e1c9b5e3f933fa35dac8bc88 ...
Feeding VM account code at 0xbbda93a81b882774e1c9b5e3f933fa35dac8bc88 ...
Feeding VM account code at 0xbbda93a81b882774e1c9b5e3f933fa35dac8bc88 ...
Feeding VM account code at 0xbbda93a81b882774e1c9b5e3f933fa35dac8bc88 ...
Feeding VM account code at 0xbbda93a81b882774e1c9b5e3f933fa35dac8bc88 ...
Feeding VM account code at 0xbbda93a81b882774e1c9b5e3f933fa35dac8bc88 ...
Feeding VM account code at 0xbbda93a81b882774e1c9b5e3f933fa35dac8bc88 ...
...
Feeding VM account code at 0xbbda93a81b882774e1c9b5e3f933fa35dac8bc88 ...
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 99, message: "Cannot assign requested address" } })', /checkout/src/libcore/result.rs:859
stack backtrace:
   0: std::sys::imp::backtrace::tracing::imp::unwind_backtrace
   1: std::sys_common::backtrace::_print
   2: std::panicking::default_hook::{{closure}}
   3: std::panicking::default_hook
   4: std::panicking::rust_panic_with_hook
   5: std::panicking::begin_panic
   6: std::panicking::begin_panic_fmt
   7: rust_begin_unwind
   8: core::panicking::panic_fmt
   9: core::result::unwrap_failed
             at /checkout/src/libcore/macros.rs:29
  10: <core::result::Result<T, E>>::unwrap
             at /checkout/src/libcore/result.rs:737
  11: gethrpc::GethRPCClient::rpc_object_request
             at /home/stewart/dev/fractalide/etcdev/sputnikvm/gethrpc/src/lib.rs:155
  12: gethrpc::GethRPCClient::rpc_request
             at /home/stewart/dev/fractalide/etcdev/sputnikvm/gethrpc/src/lib.rs:142
  13: gethrpc::GethRPCClient::get_code
             at /home/stewart/dev/fractalide/etcdev/sputnikvm/gethrpc/src/lib.rs:254
  14: regtests::handle_fire
             at ./src/bin/main.rs:100
  15: regtests::test_block
             at ./src/bin/main.rs:154
  16: regtests::main
             at ./src/bin/main.rs:259
  17: std::panicking::try::do_call
  18: __rust_maybe_catch_panic
  19: std::rt::lang_start
  20: main
  21: __libc_start_main
  22: _start
             at ../sysdeps/x86_64/start.S:120

```

**Comment by sorpaas** [comment]
@sjmackenzie This should have been fixed. Have a look.

**Comment by sjmackenzie** [comment]
still testing #191 range

**Comment by sjmackenzie** [comment]
fixed in #196
